### PR TITLE
fix:menu的token在light/dark模式下有bug

### PR DESCRIPTION
**Pull request by cheng87126**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [*] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link
https://github.com/ant-design/ant-design/issues/45488

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |add itemTitleSelectedColor for select title|
| 🇨🇳 Chinese |增加标题选中颜色参数|

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [*] Doc is updated/provided or not needed
- [*] Demo is updated/provided or not needed
- [*] TypeScript definition is updated/provided or not needed
- [*] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 4889cf6</samp>

Added a new theme option to change the selected menu item title color in `Menu` component. Modified `components/menu/style/index.tsx` and `components/menu/style/theme.tsx` to support this option.

### <samp>🤖 Generated by Copilot at 4889cf6</samp>

* Add a new theme token `itemTitleSelectedColor` to customize the color of the selected menu item title text ([link](https://github.com/ant-design/ant-design/pull/45563/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dR95-R99), [link](https://github.com/ant-design/ant-design/pull/45563/files?diff=unified&w=0#diff-f84706d0f4fb54c91e83f8dd7b34ce37f6015d56dc057385580b19473c8f1f7dR948), [link](https://github.com/ant-design/ant-design/pull/45563/files?diff=unified&w=0#diff-803a82d9b4afa684620a38d6c44836f0b24b12205ef42ec80dba78aa28fe9829R15))
* Apply the `itemTitleSelectedColor` token to the menu item title text instead of the whole menu item in the `getThemeStyle` function in `theme.tsx` ([link](https://github.com/ant-design/ant-design/pull/45563/files?diff=unified&w=0#diff-803a82d9b4afa684620a38d6c44836f0b24b12205ef42ec80dba78aa28fe9829L69-R70))

### CODE REVIEW

File: components/menu/style/theme.tsx [hunk 1]
```diff
@@ -66,7 +67,7 @@ const getThemeStyle = (token: MenuToken, themeSuffix: string): CSSInterpolation
 
       [`${componentCls}-submenu-selected`]: {
         [`> ${componentCls}-submenu-title`]: {
-          color: itemSelectedColor,
+          color: itemTitleSelectedColor,
```
Comment (afc163) [inline components/menu/style/theme.tsx:70]:
```suggestion
          color: subMenuItemSelectedColor,
```

File: components/menu/style/index.tsx [hunk 2]
```diff
@@ -92,6 +92,11 @@ export interface ComponentToken {
    * @descEN Color of selected menu item text
    */
   itemSelectedColor: string;
+  /**
+   * @desc 菜单项标题文字选中颜色
+   * @descEN Color of selected menu item title text
```
Comment (MadCcc) [inline components/menu/style/index.tsx:97]:
```suggestion
   * @descEN Color of selected sub menu item title text
```

**Comment by MadCcc** [comment]
这个可能不解决 issue 中的问题。需要把真实选中节点的和子节点被选中的父节点区分开

**Comment by cheng87126** [comment]
> 这个可能不解决 issue 中的问题。需要把真实选中节点的和子节点被选中的父节点区分开

.ant-menu-submenu-selected和.ant-menu-item-selected 不是已经做了区别吗

**Comment by afc163** [comment]
Fixed in https://github.com/ant-design/ant-design/pull/52182

**Pull request closed without merging**